CIP-?/README.md

```diff
@@ -0,0 +1,154 @@
+# CIP-?: NFT and Token Authentication
+
+---
+- **CIP**: ?
+- **Title**: NFT and Token Authentication
+- **Authors**: Emir Olgun <emirolgun@gmail.com>
+- **Discussions-To**: https://github.com/littlefish-foundation/CIPs/discussions
+- **Status**: Proposed
+- **Created**: 2024-09-17
+- **License**: CC-BY-4.0
+---
```

Comment:
```suggestion
# CIP-?: NFT and Token Authentication

---
CIP: ?
Title: NFT and Token Authentication
Authors:
- Emir Olgun <emirolgun@gmail.com>
Discussions:
- https://github.com/cardano-foundation/CIPs/pull/917
Implementors: []
Status: Proposed
Created: 2024-09-17
License: CC-BY-4.0
---
```
The most important change (besides the wrong document structure below): this is not a human readable list, but rather a YAML metadata section: [Padierfind](https://github.com/cardano-foundation/CIPs/blob/master/CIP-0001/README.md#header-preamble)